ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -636,4 +636,18 @@ FAN_SS_VSC_EPC_EHHRC_EHHRC3X:
   uses:
   - heat_recovery_water_isolation_valve_command_1
   - heat_recovery_water_isolation_valve_command_2
-  - heat_recovery_water_isolation_valve_command_3
\ No newline at end of file
+  - heat_recovery_water_isolation_valve_command_3
+  
+FAN_SS_SFVCS_SFVSC_FDPM_STC_SRC_MIPCVM_EPM_SFRDM:
```

Comment:
How can it have a `SS` and `SFVSC`? Arent they the same fan? Also, what is with all the other random control? Fans do not have filters.

Reply:
the above canonical has been deleted...However, the system is modeled as an MAU and discharges the air to a space. The equipment has filters and we monitor differential pressure across the filter.